_data/accessibility-tests/radio-buttons.yml

```diff
@@ -0,0 +1,72 @@
+title: Radio buttons
+component_status: pass
+test_items:
+# General tests
+  - summary: Focus indicator on radio button is clearly visible.
+    summary_additional: When you use a keyboard to focus on a radio button group, you will see a visible outline or other clear indication of focus.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 3.2.1
+  - summary: Radio button labels are descriptive, brief and clear.
+    summary_additional: When you view radio buttons, all labels have short descriptions that are easy to understand and act on.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 2.4.6
+  - summary: All clickable elements touch target size meet height and width requirements.
```

Comment:
Feel like this is worded a little awkwardly. Recommend:
"Touch target size for all clickable elements meets height and width requirements."
@amycole501 and @amyleadem does this still make sense for the test?

Reply:
Sounds good to me. Confirming that it aligns with what we used for the [header page](https://designsystem.digital.gov/components/header/accessibility-tests/): "The touch target on all interactive elements meets height and width requirements."

![image](https://github.com/user-attachments/assets/db81c36e-fc1c-46de-b85a-9f990f391e3e)

Reply:
Resolved in [610d52e](https://github.com/uswds/uswds-site/pull/2821/commits/610d52ee302096392b62b51d30b4551a0a179ec6)